### PR TITLE
Support `HtmlString` in repeater and builder `itemLabel`

### PR DESCRIPTION
**Pull request by bernhardh**
As discussed in PR #6191

### CODE REVIEW

**Comment by danharrin** [comment]
If an `HtmlString` object is used, you shouldn't need to make changes like this to the view. This is just opening everything up to XSS attacks.

**Comment by bernhardh** [comment]
Ok. I hope this was my last mistake ;)

File: packages/forms/src/Components/Repeater.php [hunk 1]
```diff
@@ -488,7 +488,7 @@ public function getCachedExistingRecords(): Collection
         );
     }
 
-    public function getItemLabel(string $uuid): ?string
+    public function getItemLabel(string $uuid): null | string | Htmlable
```
Comment (zepfietje) [inline packages/forms/src/Components/Repeater.php]:
```suggestion
    public function getItemLabel(string $uuid): string | Htmlable | null
```

**Review by zepfietje — APPROVED** [review]

**Comment by danharrin** [comment]
Thanks

**Pull request merged**